### DISCUSSION TITLE
Hide "vercel.app" in navigation bar

### DISCUSSION THREAD

**Discussion by Okilele-newbie** [vercel/vercel#4809] [Help] [answered]
I have a domain name  "my domain.com" redirected to "mydomain" project on Vercel. This works perfectly but I would like when somebody enters my domain.com in its navigator this remains as he entered it and not replaced by mydomain.vercel.com. 
Actually I think this is not acceptable for a production site.
Is there a solution?

**Comment by paulogdm** [comment, marked as answer]
Hello,

Please follow the instructions on this page carefully: https://vercel.com/docs/v2/custom-domains

**Reply by Okilele-newbie** [reply]
Yes, I did it.
So when entering mydomain.com I correctly get my web app. But the navigation bar shows mydomain.vercel.app.
Did I miss something?

**Reply by paulogdm** [reply]
You may have misconfigured your domain. Again, you need to pay attention to the configuration instructions and follow them accordingly.

**Reply by Okilele-newbie** [reply]
Well, I have to create DNS record. What is not clear for me is : Where should I create them. In the provider's interface where my domain name is parked or in yours. I should not be authorized in yours because I did authentify for the domain name. So I did it at my provider's. And this works as I reach your server at 76.76.21.21.
And the other hand this does not work perfectly and I was able to create a CNAME for my www subdomain!
So I'm lost.
I deleted my account to restart from scratch but it seems I wave to wait 48 hrs ...

**Reply by sauravgupta2800** [reply]
@Okilele-newbie how did you solved it?

I am also facing the same issue.

I bought a domain from 3rd party.
example www.test.com and when I open it in the browser. It changed the text in the search bar as test.vercel.app automatically


Please help me.

**Reply by junagao** [reply]
Hi @Okilele-newbie and @sauravgupta2800,

I was having the same issue and solved by changing the .vercel.app domain configuration 'Redirect to' to point to my 3rd party domain and the status code set to 308 Permanent Redirect.

Have you tried that?
